Core/src/main/java/com/plotsquared/core/configuration/caption/load/ClassLoaderCaptionProvider.java

```diff
@@ -53,7 +53,8 @@ final class ClassLoaderCaptionProvider implements DefaultCaptionProvider {
         try {
             final InputStream stream = this.classLoader.getResourceAsStream(url);
             if (stream == null) {
-                LOGGER.warn("No resource for locale '{}' found", locale);
+                LOGGER.info("No resource for locale '{}' found in the plugin file, please ensure you have placed the latest version of the file messages_{}.json in the 'lang' folder. You may be able to find completed translations at https://intellectualsites.crowdin.com/plotsquared",
```

Comment:
Please put a few line breaks here to make the code easier to read

Reply:
Done.